MAINTAINERS.md

```diff
@@ -0,0 +1,101 @@
+# Kubernetes Project Maintainership
+
+Kubernetes is a big project with many contributors, the project organization is
+a daunting task and the project maintainers are the members that are trusted
+with that task, driving the project, improving the health of the project,
```

Comment:
We'll want to firm up these responsibilities, over and above expectations of all contributors, which we should link to here:
https://github.com/kubernetes/kubernetes/blob/master/docs/devel/community-expectations.md